workerapi/server/server.go

```diff
@@ -13,8 +13,24 @@ import (
 	"github.com/scootdev/scoot/workerapi/gen-go/worker"
 )
 
+// Creates a Worker Server
+func MakeServer(
+	handler worker.Worker,
+	transport thrift.TServerTransport,
+	transportFactory thrift.TTransportFactory,
+	protocolFactory thrift.TProtocolFactory) thrift.TServer {
+	return thrift.NewTSimpleServer4(
+		worker.NewWorkerProcessor(handler),
+		transport,
+		transportFactory,
+		protocolFactory)
+}
+
 // Called by a main binary. Blocks until the connection is terminated.
-func Serve(handler worker.Worker, addr string, transportFactory thrift.TTransportFactory, protocolFactory thrift.TProtocolFactory) error {
+func Serve(
```

Comment:
it doesn't right now, but closed source does still, so deleting this now would cause a breaking change.

Until I merge this and change that code base I can't delete this yet.  There will be a follow up pull request removing this when all of the changes in Twitter Source are done.